description.md

```diff
@@ -17,6 +17,86 @@ Each section details how a specific type of content is to be developed, stored a
 
 ### Quizzes
 
+#### Definition
+
+Quizzes are questions that verify specific information and must be verifiable automatically.
+Therefore, multiple-choice questions are regarded as quizzes, while open-ended ones are not.
+Quizzes may be part of practical sessions, lectures or standalone as forms of evaluation.
+
+#### Purpose
+
+Quizzes provide a means to continuously evaluate students.
+This creates a positive feedback loop between trainers and trainees whereby both can track the students' progress in real time.
+
+Furthermore, quizzes serve as practice items for students to solve at their own pace.
+They may also be used for evaluation items that contribute to the students' overall grade.
+
```

Comment:
For completeness, I think it is worth mentioning the downsides of quizzes:

- not suitable for some fields of study
- binary feedback loop: correct/false
- maybe others?